src/commands/deployment/README.md

```diff
@@ -56,6 +56,7 @@ datadog-ci deployment correlate --provider argocd
 
 - `--provider` (**required**): the CD provider name. Currently, the only supported CD provider is `argocd`.
 - `--config-repo`: configuration repository URL where the kubernetes manifests are stored. If empty, the command tries to get it using the git command `git ls-remote --get-url`.
+- `--config-shas`: a list of the Git commit SHAs of the configuration repository. If empty, the command tries to get all unpushed commits using a `git log` command.
```

Comment:
We also need to take a look at the rest of the docs. For example above we state:

> In order for the command to work properly, it needs to be called between when the configuration changes are committed and when they are pushed to the configuration repository (where the Kubernetes manifests are).

This is now one of the use cases we will support. If config-shas are not specified (and config-repo as well??), they need to execute it at a specific point. We should adapt the docs to mention that commits detection only works for the commit-push use case if the command is executed in the middle. For other use cases, they should explicitly define the shas (and repo?)